llvm/include/llvm/SYCLLowerIR/ModuleSplitter.h

```diff
@@ -270,6 +273,33 @@ void dumpEntryPoints(const Module &M, bool OnlyKernelsAreEntryPoints = false,
                      const char *msg = "", int Tab = 0);
 #endif // NDEBUG
 
+struct SplitModule {
```

Comment:
Why do we need this new data structure? Can we not continue to use IrPropSymFilenameTriple?

Thanks

Reply:
This is possible.